packages/kbn-generate-csv/src/lib/search_cursor_pit.ts

```diff
@@ -37,7 +37,7 @@ export class SearchCursorPit extends SearchCursor {
     this.cursorId = await this.openPointInTime();
   }
 
-  private async openPointInTime() {
```

Comment:
A lot of changes in this PR are to use Test classes (`TestSearchCursorPit` and `TestSearchCursorScroll`) that extend protected methods in the original classes. This was done to remove the `@ts-expect-errors` lines of code.